create-object-store/action.yml

```
@@ -0,0 +1,35 @@
+name: "Create Object Store"
+description: "Create Object Store"
+inputs:
+  api-token:
+    description: "API Token."
+    required: true
+    default: ""
+  cluster-id:
+    description: "Cluster id to attach object store to"
+    required: true
+    default: ""
+  bucket-name:
+    description: "Name of the bucket to create"
+    required: true
+    default: ""
```

We should be consistent with the cli for all inputs

```suggestion
  bucket-prefix:
    description: "A prefix for the bucket name to be created"
    required: true
    default: ""
```

---

The cli uses `bucket` as flag